app/src/main/java/com/artemzin/qualitymatters/ui/presenters/Presenter.java

```diff
@@ -48,7 +48,7 @@ protected final void unsubscribeOnUnbindView(@NonNull Subscription subscription,
     public void unbindView(@NonNull V view) {
         final V previousView = this.view;
 
-        if (previousView == view) {
+        if (view.equals(previousView)) {
```

Comment:
`==` was here for a reason. Two views displaying same data are different from the point of Presenter.

Please change it back to `==` :) I guess, you'll need to add `@SupressWarnings("SomePmdCheck")`.